docs/readiness-probes.md

```diff
@@ -40,30 +40,30 @@ containers:
     httpGet:
       path: /v1/models
       port: 8000
-    initialDelaySeconds: 15    # Time before first probe
-    periodSeconds: 30           # How often to probe during startup
-    timeoutSeconds: 5           # HTTP request timeout
-    failureThreshold: 60        # Max attempts (30s * 60 = 30min max startup time)
+    initialDelaySeconds: 5    # Time before first probe (reduced for faster response)
+    periodSeconds: 5         # How often to probe during startup (increased frequency)
```

Comment:
There's no disadvantage to probing an HTTP endpoint that is cheap every 1 second.

Also, the startup probe should be on the health endpoint because the model server doesn't start unless the models are loaded, and health is cheap.